src/plus/gk/account/subscriptionService.ts

```diff
@@ -359,14 +361,41 @@ export class SubscriptionService implements Disposable {
 			);
 		}
 
+		return this.loginCore({ signUp: signUp, source: source });
+	}
+
+	async loginWithCode(code: string, state?: string, source?: Source): Promise<boolean> {
```

Comment:
Combine `code` & `state` into an `authentication?: { code: string; state?: string }` argument (I can't think of a better name for the arg as `signIn` like the others doesn't really make sense here)